### ISSUE TITLE
Server Unplayable

### ISSUE THREAD

**Issue by kingkj52** [GreatOrator/TolkienCraftII#59]
Are there settings that need to be changed to host this? My friend was able to host Infinity servers with no lag whatsoever, but when joining on TolkienCraft 2, there are on average 30 seconds of delay between when he sends a message and when I receive it. I can send a message, and he'll see it instantly, but I won't see my own sent message for ~30 seconds. Anything I do (talking to Elmn, for example) is also delayed by a huge amount.

Another issue was the inability to start for him (get a quest book/enable quest mode) though I think it was because he didn't enable command blocks. 

tl;dr, are there special settings that need to be applied for hosting tolkiencraft?

**Comment by GreatOrator** [comment]
This modpack is actually much heavier than most and SSP requires 3-3.5GB of memory.  So your friend running the pack AND hosting a server is most likely too much for his pc to do on it's own.  Command blocks are important, but you seem to have noticed that.  I can only suggest if you want a server to look at a server host specifically to help with that.  Resonant Host is one that comes to mind :D

**Comment by kingkj52** [comment]
He just built his computer for $3k a couple months ago: 16 GB DDR3 Ram, 4790k, two 980 TIs in SLI, I thought it would be enough, but maybe not xD. Thanks for the help, I'll see if there's any optimizing with the start batch I can do on our end.

**Comment by GreatOrator** [comment]
Problem is with minecraft the video has almost no effect, it is all memory and processor because java

**Comment by kingkj52** [comment]
Yeah, I know the graphics card is irrelevant, sorry for typing it haha. Sounded sorta scummy. So even if he allocates 6GB of RAM to the server (which he has done before, with resulting lag still) its still not plausible? I used Resonant Host before but they shared the machine with others, so it lagged on infinity really hard because the resources were not all allocated to my server, but rather divided between the (3/4/5?) on the one machine. He refuses to pay for a host, and I'm too broke as a college student to do it. We'll just have to play SSP.

Thanks for the help! Hope you had happy holidays!
